scripts/deleteAndLoadSnapsets.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/sh
 APP_DIR=${APP_DIR:-"/app"}
```

Comment:
Shouldn't APP_DIR be GPII_APP_DIR?

Reply:
Hm, there's that one (and 2 or 3 more in the script). I don't have a strong opinion - basically I left that one out as I don't expect that one to be ever modified and prefixed all the 4 ones which are documented in the docs and used in the `vagrantCloudBasedContainers.sh` script.

Reply:
I don't have a strong opinion either.  One differences is that this one is an environment variable, whereas the rest are "local".

Reply:
What do you mean by "local"? I'm not sure how is this one different, and none of those vars is exported.

Reply:
I mean that `APP_DIR` could be set as an environment variable outside of the `deleteAndLoadSnapsets.sh` script such that, when run, it uses that definition rather than the default `"/app"`.  But variables like `DATALOADER_JS` are defined within the script.  Even in the unlikely event that there is an environmental variable with the same name, it would be overridden.

Put another way, `APP_DIR` uses conditional assignment where if it is defined outside the script, the script uses that outside definition.  But, DATALOADER_JS is set unconditionally within the script.

Reply:
OK, thanks, I understand what you meant, I've added the prefix to he APP_DIR as well.

Reply:
Thanks -- pulling these changes in.